aws/monolith-terraform/xebialabs/USAGE.md.tmpl

```diff
@@ -8,15 +8,11 @@ The application is available in the repo [https://github.com/xebialabs/e-commerc
 
 ### Prerequisites
 
-If you do not have XL Release and XL Deploy instances running, you can use this Docker Compose setup to spin them up. The setup also contains a preconfigured Jenkins instance.
-Please note that you need to obtain a trial license or bring your own license for XL Release and XL Deploy. Follow the steps below to use the Docker Compose files.
+If you do not have XL Release and XL Deploy instances running, you can use the Docker Compose setup generated by the blueprint to spin them up.
+Please note that you can obtain a trial license or bring your own license for XL Release and XL Deploy if you like. Follow the steps below to use the Docker Compose files.
```

Comment:
Maybe we could explain how to apply automatically their own license? They can use the `default-conf` folder right?

Reply:
not anymore, seems like you need to set them in env variable and stuff. Maybe I can point to some docs on how to add licence

Reply:
seems like there is no clear docs on licence installtion for docker and with Jetpack and stuff its even more confusing so lets leave it out here

Reply:
shall we merge, then?

Reply:
The jetpack pages also doesnt say about licence. Anyways I think when licence expires you can add it via UI